apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GrpcLogConsumer.cs

```diff
@@ -43,7 +45,18 @@ public void Receive(IEnumerable<LogEntry> logs)
             {
                 return;
             }
-            _client.WriteLogEntriesAsync(null, null, EmptyDictionary, logs);
+            _client.WriteLogEntries(null, null, EmptyDictionary, logs);
+        }
+
+        /// <inheritdoc />
+        public async Task ReceiveAsync(
+            IEnumerable<LogEntry> logs, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (!logs.Any())
+            {
+                return;
+            }
+            await _client.WriteLogEntriesAsync(null, null, EmptyDictionary, logs, cancellationToken);
```

Comment:
Removing 'async' on this one will also require updating the return above to return a completed task, by the way.